### PR TITLE
traefik: add page

### LINKED ISSUE #1184: page request: traefik
An HTTP reverse proxy that natively supports container technology (Docker, Docker Swarm, Kubernetes)

https://github.com/containous/traefik

### PR DESCRIPTION
**Pull request by blazedd**
Closes #1184

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/traefik.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> Traefik HTTP reverse proxy and load balance.
```
Comment (agnivade) [inline pages/common/traefik.md]: No need to mention traefik again in the heading. Also typo in balance.

I suggest - `A HTTP reverse proxy and load balancer`

File: pages/common/traefik.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Start server with default config:
+
+`traefik`
+
+- Start server with custom config file:
+
+`traefik --c {{config_file}}.toml`
+
+- Start server with a cluster mode enabled:
```
Comment (agnivade) [inline pages/common/traefik.md]: remove "a"

File: pages/common/traefik.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`traefik --cluster`
+
+- Start server with web ui enabled:
```
Comment (agnivade) [inline pages/common/traefik.md]: ui => UI.

Also, the words "Start server" are kinda getting repetitive. Maybe you can just say "Enable web UI".

File: pages/common/traefik.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> A HTTP reverse proxy and load balancer.
+
+- Start server with default config:
```
Comment (sbrl) [inline pages/common/traefik.md]: Perhaps making it "traefik server" instead of just server? Same with below.

File: pages/common/traefik.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+`traefik`
+
+- Start server with custom config file:
```
Comment (sbrl) [inline pages/common/traefik.md]: `with custom config file` -> `with a custom config file` might read better.

File: pages/common/traefik.md (hunk 4 above)
Comment (blazedd) [reply pages/common/traefik.md]: I was merely matching the same wording used on the similar nginx page. I can adjust it if you'd like...

File: pages/common/traefik.md (hunk 5 above)
Comment (blazedd) [reply pages/common/traefik.md]: As I stated in your other change, this was mirror to the Nginx page's wording. I'm open to changing it if you feel it's necessary.

File: pages/common/traefik.md (hunk 4 above)
Comment (sbrl) [reply pages/common/traefik.md]: Ah, ok! Let's leave it then :smiley_cat:

File: pages/common/traefik.md (hunk 5 above)
Comment (sbrl) [reply pages/common/traefik.md]: Here it reads as better English if it has an article before the noun "config file",  so I think this change is still needed. I'll update the nginx page too, if everyone else is ok with this?

**Comment by blazedd** [comment]
I've applied all requested grammar fixes to my pull request. Thanks for the feedback.

**Comment by agnivade** [comment]
@blazedd - How about this comment of mine - 
> Also, the words "Start server" are kinda getting repetitive. Maybe you can just say "Enable web UI".

File: pages/common/traefik.md (hunk 3 above)
Comment (blazedd) [reply pages/common/traefik.md]: 1. I think it's important to dictate that the flag I'm referring to, both starts the Traefik server and enables the UI. While it's true that all of my items start the server, it's worth ensuring there is no confusion of altering a configuration to enable the web UI. It's a pretty short list, but a couple flags do not actually start the server.
2. This mirrors the same language on the nginx page. Nginx is pretty similar in terms of having a small number of args that don't start the server.

Comment (agnivade) [reply pages/common/traefik.md]: >  It's a pretty short list, but a couple flags do not actually start the server.

I see. Okay then.

**Pull request merged**